PySpice/Spice/HSpice/hspicefile/docs/tests/verify_all_types.py

```
@@ -0,0 +1,1367 @@
+# ---------------------------------------------------------------------------
+# ASCII golden reference parser
+# ---------------------------------------------------------------------------
+
+
+def parse_ascii_to_flat_floats(filename):
+    """
+    Extract every 13-character float from the post-terminator data section.
+    """
+    if not os.path.exists(filename):
+        return None
+    with open(filename, "r", errors="ignore") as f:
```

Ruff (0.15.20)

[warning] Unnecessary mode argument

Remove mode argument

(UP015)